testhelpers/core.go

```diff
@@ -184,3 +187,158 @@ func RequireCreateStorages(ctx context.Context, t *testing.T) (state.Tree, vm.St
 
 	return st, vms
 }
+
+type testStorage struct {
```

Comment:
I would prefer that code like this live in the package alongside the things its faking, `vm/testing.go`. `testhelpers` is a dumping ground that does not make things easy to find and is not a scalable pattern.